lib/api/bucketPut.js

```diff
@@ -54,6 +57,12 @@ function checkLocationConstraint(request, locationConstraint, log) {
         return { error: errors.InvalidLocationConstraint.
             customizeDescription(errMsg) };
     }
+    if (locationConstraintAddon) {
+        // We are introducing a new format for location constraints.
+        // with `zenkoSeparator`, we are currently including the ingestion
+        // flag in the location constraint.
+        locationConstraintChecked = locationConstraint;
```

Comment:
this block might not be not be needed. I only see `locationConstraintAddon` being set on L#41 and `locationConstraintChecked` is already set to whatever the location name is regardless if there is a split or not

Reply:
I need this block because `locationConstraintChecked` needs to include any other values that was appended on the `locationConstraint` string, and not just the location name. The value is used later on in `createBucket`.

Reply:
ah I see, sorry misread